.github/workflows/main.yml

```
@@ -1,35 +1,27 @@
-name: Node.js CI
+name: Go Build & Test
 
 on:
   push:
-    branches: ["master", "develop"]
+    branches: ["main"]
```

Só ve se a sintax funciona assim, não precisa tá num array se só tem 1 valor

```suggestion
    branches: 
      - main
```